Makefile

```diff
@@ -80,7 +80,7 @@ unpacked:
 	$(CC) -I. mongoose.c test/unit_test.c -o $@
 
 fuzzer: mongoose.c mongoose.h Makefile test/fuzz.c
-	clang++ mongoose.c test/fuzz.c $(WARN) $(INCS) $(TFLAGS) -DMG_ENABLE_LINES -fsanitize=fuzzer,signed-integer-overflow,address -Wno-deprecated -Wno-vla-extension -o $@
+	$(CXX) test/fuzz.c $(WARN) $(INCS) $(TFLAGS) -fsanitize=fuzzer,signed-integer-overflow,address -Wno-deprecated -Wno-vla-extension -o $@
```

Comment:
Building the fuzzer fails, there are lots of "already defined" errors between fuzz.c and mongoose.o

Reply:
Yep, sent an issue to the oss-fuzz project and asked to change.
https://github.com/google/oss-fuzz/issues/8549